### ISSUE TITLE
send to repeat ，the proto is wrong（http/https）

### ISSUE THREAD

**Issue by AspirePig** [Quitten/Autorize#67]
```class sendRequestRepeater(ActionListener):
    def __init__(self, extender, callbacks, original):
        self._extender = extender
        self._callbacks = callbacks
        self.original = original

    def actionPerformed(self, e):
        if self.original:
                request = self._extender._currentlyDisplayedItem._originalrequestResponse
        else:
                request = self._extender._currentlyDisplayedItem._requestResponse
        host = request.getHttpService().getHost()
        port = request.getHttpService().getPort()
         ##### add code
        proto = request.getHttpService().getProtocol()
        secure = True if proto == 'https' else False
        
         ########  change code
        self._callbacks.sendToRepeater(host, port, secure , request.getRequest(), "Autorize");
```

**Comment by AspirePig** [comment]
class sendRequestRepeater(ActionListener):
    def __init__(self, extender, callbacks, original):
        self._extender = extender
        self._callbacks = callbacks
        self.original = original

    def actionPerformed(self, e):
        if self.original:
                request = self._extender._currentlyDisplayedItem._originalrequestResponse
        else:
                request = self._extender._currentlyDisplayedItem._requestResponse
        host = request.getHttpService().getHost()
        port = request.getHttpService().getPort()
        proto = request.getHttpService().getProtocol()
        secure = True if proto == 'https' else False
        
        self._callbacks.sendToRepeater(host, port, secure , request.getRequest(), "Autorize");

**Comment by Quitten** [comment]
@AspirePig - Thank you very much for the bug report! Next time please create a PR and I will merge it.
Fixed on bad217b9dc87476ca80313acd997536a60cc5e8f.
